### ISSUE TITLE
autoprefixer detects "transition-timing-function: cubic-bezier(...)" as complicated transition case

### ISSUE THREAD

**Issue by m-blank** [postcss/autoprefixer#621]
When running autoprefixer within my gulp build process, it displays the following message:

`../css/main.css
4653:5  ‼  Replace transition-property to transition, because Autoprefixer could not support any cases of transition-property and other transition-* [autoprefixer]`

The message gets triggered when I'm using the CSS property `transition-timing-function` with the value `cubic-bezier(0.55, 0, 0.1, 1)`. The method `Transition.prototype.checkForWarning` in /lib/transition.coffee detects this as a complicated transition case, because it looks for the occurence of a comma in [line 67](https://github.com/postcss/autoprefixer/blob/master/lib/transition.coffee#L67).

It shows this message, because the value of `transition-timing-function` contains a comma seperated list of 4 points for the cubic bezier curve. Is it possible to adjust the check?

**Comment by ai** [comment]
Oops 😀. Thanks for this tricky case. I will try to fix it in next 3 days.

**Comment by m-blank** [comment]
Thanks! :+1:
One thing that comes into my mind is extending the check with a regex that should not match the definition of a single cubic bezier curve. Something like the following:

```javascript
if ((i.value.indexOf(',') !== -1) && !/^cubic\-bezier\((\w+,){3}\w+\)$/.test(i.value)) {
    [...]
}
```

**Comment by ai** [comment]
Done eac54bb

**Comment by m-blank** [comment]
Awesome, thank you for the fast fix!

**Comment by ai** [comment]
Released in 6.3.3.
